package.json

```diff
@@ -16,10 +19,11 @@
     "clean": "gts clean",
     "codecov": "c8 report --reporter=json && codecov -f coverage/*.json",
     "fix": "gts fix",
-    "compile": "tsc -p .",
+    "compile": "tsc --target ES5 --module CommonJS",
```

Comment:
To remove `allowSyntheticDefaultImports` option I directly transpiled typescript files to ES5 and CommonJS for running tests in node.